src/wallet/rpcwallet.cpp

```diff
@@ -2315,7 +2315,7 @@ static UniValue settxfee(const JSONRPCRequest& request)
     CAmount nAmount = AmountFromValue(request.params[0]);
     CFeeRate tx_fee_rate(nAmount, 1000);
     CFeeRate max_tx_fee_rate(pwallet->m_default_max_tx_fee, 1000);
-    if (tx_fee_rate == 0) {
```

Comment:
is there a reason why we don't have the `p2p_feefilter.py` test?

Reply:
we don't have `feefilter` message